web/app/themes/dp-timber-theme/.storybook/main.js

```diff
@@ -56,7 +56,7 @@ const config = {
     });
     const twigAlias = {
       "@atoms": path.resolve(__dirname, '../', 'components/atoms'),
-      "@molecules": path.resolve(__dirname, '../', 'components/molecules'),
+      "@molecules": path.resolve(__dirname, '../', 'components/molecule'),
```

Comment:
Is because I am unable to change the Name of the folder, some restrictions are there

![image](https://github.com/shayaanmiyy/wp-starter/assets/69762513/8ea77ee1-c44c-4e05-b5ca-0e02bd89cb61)

![image](https://github.com/shayaanmiyy/wp-starter/assets/69762513/b8f72e3d-1f9a-45f8-8d10-704431a64e87)